src/include/platform/internal/GenericPlatformManagerImpl_POSIX.cpp

```diff
@@ -138,6 +139,9 @@ void GenericPlatformManagerImpl_POSIX<ImplClass>::SysUpdate()
         InetLayer.PrepareSelect(mMaxFd, &mReadSet, &mWriteSet, &mErrorSet, mNextTimeout);
     }
 #endif // !(CHIP_SYSTEM_CONFIG_USE_NETWORK_FRAMEWORK)
+#if CHIP_ENABLE_MDNS
```

Comment:
this feature flag is disabled for cirque?  is there an issue covering removal of this feature?

Reply:
https://github.com/project-chip/connectedhomeip/issues/3433